<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clsSelectConnetion</class>
 <widget class="QDialog" name="clsSelectConnetion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>86</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择连接方式</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myIcon.qrc">
    <normaloff>:/Icon/Icons/wk.ico</normaloff>:/Icon/Icons/wk.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog
{
	
	font: 75 12pt &quot;楷体&quot;;
	background-color: rgb(148, 148, 248);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="rBtnGPIB">
       <property name="styleSheet">
        <string notr="true">font: 75 16pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>GPIB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rBtnLAN">
       <property name="styleSheet">
        <string notr="true">font: 75 16pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>LAN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rBtnUSB">
       <property name="styleSheet">
        <string notr="true">font: 75 16pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>USB</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>33</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>地址：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtAddress">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbModel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>28</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
       </property>
       <item>
        <property name="text">
         <string>6500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3260</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnConnct">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myIcon.qrc"/>
 </resources>
 <connections/>
</ui>
